src/reitit/core.cljc

```diff
@@ -60,6 +61,27 @@
   (cond->> (->> (walk data opts) (map-meta merge-meta))
            coerce (into [] (keep #(coerce % opts)))))
 
+(defn conflicting-routes [routes]
```

Comment:
This whole function might be more efficient and easier to understand with transducers. 😎